mesonbuild/interpreterbase/decorators.py

```diff
@@ -290,22 +290,30 @@ class KwargInfo(T.Generic[_T]):
         different type. This is intended for cases such as the meson DSL using a
         string, but the implementation using an Enum. This should not do
         validation, just converstion.
+    :param deprecated_values: a dictionary mapping a value to the version of
+        meson it was added in.
```

Comment:
should this comment read 'deprecate in' rather than 'added in'?

Reply:
yup, it should. fixed the in the latest vresion